packages/push/src/PushRegistration.ts

```diff
@@ -78,13 +95,29 @@ export class PushRegistration {
       "alias": alias,
       "categories": categories
     };
+
     const instance = axios.create({
       baseURL: url,
       timeout: 5000,
-      headers: { "Authorization": `Basic ${authToken}` }
+      headers: {"Authorization": `Basic ${authToken}`}
     });
+
     return new Promise((resolve, reject) => {
-      return instance.post(PushRegistration.API_PATH, postData).then(() => resolve()).catch(reject);
+      return instance.post(PushRegistration.API_PATH, postData)
+      .then(
+        () => {
+          if (isCordovaAndroid()) {
+            this.subscribeToFirebaseTopic(variantId);
+            for (const category of categories) {
+              this.subscribeToFirebaseTopic(category);
+            }
+            resolve();
```

Comment:
A slightly nicer and more readable way to structure that would be as follows

```
if (isCordovaAndroid()) {
  // do the things
}
resolve()
```

there isn't really any need for the else statement and generally, it's not a great pattern to have multiple `resolve()` calls in the code because it can be confusing.

Reply:
Done!